vue-frontend/src/components/home-new/ServiceSectionMobile.vue

```diff
@@ -0,0 +1,117 @@
+<template>
+  <div
+    class="tw-text-center tw-container tw-mt-20 tw-m-0 tw-p-0 tw-max-w-full tw-relative"
+  >
+    <div
+      class="tw-flex tw-justify-center sm:tw-py-12 tw-py-9 tw-font-roboto-medium tw-leading-[4.46875rem] sm:tw-text-[3.4375rem] tw-text-[2.188rem]"
```

Comment:
Let's use 3 variants of `text-size` and `line-height` everywhere as we have used in App.vue. For lg, md and sm